### PR TITLE
Migrate null safety

### PR DESCRIPTION
**Pull request by daisuke310vvv**

### CODE REVIEW

**Comment by Faiyyazauxy** [comment]
Please merge this branch since we are not able to run it on flutter 2.0

File: packages/youtube_player_flutter/lib/src/player/raw_youtube_player.dart [hunk 1]
```diff
@@ -377,9 +372,9 @@ class _RawYoutubePlayerState extends State<RawYoutubePlayer>
     </html>
   ''';
 
-  String boolean({@required bool value}) => value ? "'1'" : "'0'";
+  String boolean({required bool value}) => value ? "'1'" : "'0'";
 
-  String get userAgent => controller.flags.forceHD
+  String? get userAgent => controller!.flags.forceHD
       ? 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.90 Safari/537.36'
       : null;
```
[the remark this answers is not in the document]
Comment (vkmaxcooldude) [reply packages/youtube_player_flutter/lib/src/player/raw_youtube_player.dart:379]: True

**Comment by GuimG** [comment]
Please merge

**Comment by RR-Reddy** [comment]
To use these changes please use dependency as below


```
  youtube_player_flutter: 
    git:
      url: https://github.com/RR-Reddy/youtube_player_flutter.git
      ref: migrate-null-safety
      path: packages/youtube_player_flutter
```


we have do below changes because of inappwebveiw plugin uses below versions 
 **com.android.tools.build:gradle:4.0.1** in build.gradle file
**distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip** in gradle-wrapper.properties file

**Comment by sarbagyastha** [comment]
Closing as already merged #464 . Thank you for the PR.

**Pull request closed without merging**